iree/samples/simple_embedding/BUILD

```diff
@@ -14,25 +14,53 @@
 
 load("//iree:build_defs.oss.bzl", "iree_cmake_extra_content")
 load("//iree/tools:compilation.bzl", "iree_bytecode_module")
+load("//build_tools/embed_data:build_defs.bzl", "c_embed_data")
 
 package(
     default_visibility = ["//visibility:public"],
     features = ["layering_check"],
     licenses = ["notice"],  # Apache 2.0
 )
 
+c_embed_data(
+    name = "simple_embedding_test_llvm_aot_rv64",
+    srcs = [
+        "data/simple_embedding_test-llvm-aot_rv64.vmfb",
+    ],
+    c_file_output = "simple_embedding_test_llvm_aot_rv64.c",
+    flatten = True,
+    h_file_output = "simple_embedding_test_llvm_aot_rv64.h",
+)
+
+cc_binary(
+    name = "simple_embedding_run",
+    srcs = ["simple_embedding_run.c"],
```

Comment:
For https://github.com/google/iree/issues/5481, I'll want to delete `simple_embedding_test` if this `simple_embedding_run` demonstrates enough of the same things. We can leave that for follow-up work.

Reply:
It's queued up in the next PR. :) I already have the commit ready.